js/src/3rdparty/sms-verification/index.js

```diff
@@ -15,17 +15,6 @@
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
 import { stringify } from 'querystring';
-import React from 'react';
-
-export const termsOfService = (
```

Comment:
I guess we have three choices.

1. not bundling the legal text with the library
2. using sth like [`dangerouslySetInnerHTML`](https://facebook.github.io/react/docs/dom-elements.html#dangerouslysetinnerhtml).
3. using plain text legal terms. they look less nice and the links are gone, but we still get rid of React.